include/disccord/rest/api_client.hpp

```diff
@@ -184,9 +185,9 @@ namespace disccord
 
                     pplx::task<void> delete_guild_role(uint64_t guild_id, uint64_t role_id, const pplx::cancellation_token& token = pplx::cancellation_token::none());
 
-                    //TODO: get_guild_prune_count
+                    pplx::task<web::json::value> get_guild_prune_count(uint64_t guild_id, uint32_t days = 1, const pplx::cancellation_token& token = pplx::cancellation_token::none());
 
-                    //TODO: begin_guild_prune
+                    pplx::task<web::json::value> begin_guild_prune(uint64_t guild_id, uint32_t days = 1, const pplx::cancellation_token& token = pplx::cancellation_token::none());
```

Comment:
These two events both return the same response, so it would probably be best to use a model for it.